src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,62 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $day-gap;
```

Comment:
The variable `$day-gap` is used here, but it's not defined within this file. Ensure that this variable is defined in a shared variables file or within this file to avoid compilation errors.